homeassistant/components/weatherflow/event.py

```diff
@@ -0,0 +1,104 @@
+"""Event entities for the WeatherFlow integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from pyweatherflowudp.device import EVENT_RAIN_START, EVENT_STRIKE, WeatherFlowDevice
+
+from homeassistant.components.event import EventEntity, EventEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from .const import DOMAIN, LOGGER, format_dispatch_call
+
+
+@dataclass(frozen=True, kw_only=True)
+class WeatherFlowEventEntityDescription(EventEntityDescription):
+    """Describes a WeatherFlow event entity."""
+
+    wf_event: str
+    event_types: list[str]
+
+
+EVENT_DESCRIPTIONS: list[WeatherFlowEventEntityDescription] = [
+    WeatherFlowEventEntityDescription(
+        key="precip_start_event",
+        translation_key="precip_start_event",
+        event_types=["precip_start"],
```

Comment:
```suggestion
        event_types=["precipitation_start"],
```
I think this would make more sense